jwst/pipeline/calwebb_spec3.py

```diff
@@ -211,10 +214,16 @@ def process(self, input):
 
                 if exptype in ['NIS_SOSS']:
                     # For NIRISS SOSS, don't save the extract_1d results,
-                    # they're identical to the calwebb_spec2 x1d products
+                    # instead run photom on the extract_1d results and save
+                    # those instead.
                     self.extract_1d.save_results = False
+                    self.photom.save_results = self.save_results
```

Comment:
Should we also be turning off save_results for extract_1d in the calwebb_spec2 pipeline for the case of SOSS data? Or is that already included and I just missed it?

Reply:
Yeah I missed this in the first implementation. Should be done properly now, pending test results should be good for another review.